### ISSUE TITLE
Faire une page dédiée pour les speakers 

### ISSUE THREAD

**Issue by jefBinomed** [GDG-Nantes/devFestSite-2016#54]
Le but de cette page est d'offrir aux speakers une page avec tous les éléments dont ils auraient besoin le jour j : slides / résolution / ...
Cette page ne sera pas référencé mais sera accessible sur le net.

En gros c'est plus pratique que de retrouver un mail

**Comment by jlandure** [comment]
On pourrait aussi le rajouter sur le "disclaimer" sur CFP...

**Comment by jefBinomed** [comment]
Yep en effet, en fait l'idée vient du gdg et avec Pierrick on trouvait que c'était une bonne idée, l'avantage c'est qu'on est pas pressé pour cette page 😃
